pulp_rpm/tests/functional/api/test_sync.py

```diff
@@ -453,11 +455,11 @@ def test_optimize(
 
     assert any(report.code == "sync.was_skipped" for report in task.progress_reports)
 
-    # sync again with sync_policy='mirror_complete', assert not optimized, but repository
-    # version is unchanged
+    # sync again with sync_policy='mirror_content_only', assert not optimized, and the latest
+    # repository version has not changed
     first_sync_repo_version = rpm_repository_api.read(repository.pulp_href).latest_version_href
     repository_sync_data = RpmRepositorySyncURL(
-        remote=new_remote.pulp_href, sync_policy="mirror_complete"
+        remote=new_remote.pulp_href, sync_policy="mirror_content_only"
```

Comment:
This one needs to stay as mirror_complete, just the previous syncs (which don't specify, and therefore default to additive) ought to be changed.

Reply:
Ah, oh, Let me do it once again.